example/android/app/src/main/kotlin/io/maido/intercomexample/MainActivity.kt

```diff
@@ -1,13 +1,6 @@
 package io.maido.intercomexample
 
-import android.os.Bundle
+import io.flutter.embedding.android.FlutterActivity
 
-import io.flutter.app.FlutterActivity
-import io.flutter.plugins.GeneratedPluginRegistrant
-
-class MainActivity(): FlutterActivity() {
-  override fun onCreate(savedInstanceState: Bundle?) {
-    super.onCreate(savedInstanceState)
-    GeneratedPluginRegistrant.registerWith(this)
-  }
+class MainActivity() : FlutterActivity() {
 }
```

Comment:
This file can therefore be removed.

Reply:
@Zazo032 no need to delete this file. As in the future if we may have to add some code here.